src/vscode-utils.ts

```diff
@@ -12,7 +14,8 @@ export interface IPackageInfo {
 
 export function getPackageInfo(context: vscode.ExtensionContext): IPackageInfo {
     const metadataFile: string = "package.json";
-    const metadata = require(path.join(context.extensionPath, metadataFile));
```

Comment:
After switching to `webpack`, the `require` is only for module inclusion, and for reading a file, we should use more proper APIs to achieve it. Otherwise, `webpack` will treat the path as a module and complains it cannot find it at runtime.